pom.xml

```diff
@@ -129,6 +129,9 @@
         <spark.archive.name>spark-${spark.version}-bin-hadoop${hadoop.binary.version}.tgz</spark.archive.name>
         <spark.archive.mirror>https://archive.apache.org/dist/spark/spark-${spark.version}</spark.archive.mirror>
         <spark.archive.download.skip>false</spark.archive.download.skip>
+        <swagger.version>2.1.11</swagger.version>
+        <swagger.scala.module.version>2.5.2</swagger.scala.module.version>
+        <swagger-ui.version>3.52.1</swagger-ui.version>
```

Comment:
I'm curious why we do not use 4.x version ?

Reply:
updated.